index.js

```diff
@@ -40,8 +46,10 @@ const inputs = [
         validator: v => v.match(/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i)
     }),
     read({
-        prompt: 'Domain that the cert will be issued for, e.g. example.com:',
-        argument: 'domain'
+        prompt: 'Domain(s) that the cert will be issued for (separated by spaces), e.g. example.com www.example.com foo.example.com:',
+        argument: 'domain',
+        processValue: v => v === undefined || Array.isArray(v) ? v : v.trim().split(/\s+/),
+        validator: v => v !== undefined && v.length > 0 && v.every(item => typeof item === 'string' && item.length > 0)
```

Comment:
Thanks for the clarification. I'm starting to think that making this an interactive tool is not worth the complexity tradeoff, and we should just force all options to be passed directly. This is what everyone should do if GitLab creates a cert API anyway, to allow automation.